extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/KafkaRecordSupplier.java

```diff
@@ -193,15 +191,12 @@ public static void addConsumerPropertiesFromConfig(
 
   private KafkaConsumer<byte[], byte[]> getKafkaConsumer()
   {
+    final Map<String, Object> consumerConfigs = KafkaConsumerConfigs.getConsumerProperties();
     final Properties props = new Properties();
-
-    props.setProperty("metadata.max.age.ms", "10000");
-    props.setProperty("group.id", StringUtils.format("kafka-supervisor-%s", RandomIdUtils.getRandomId()));
+    props.putAll(consumerConfigs);
```

Comment:
This should be called after KafkaRecordSupplier.addConsumerPropertiesFromConfig

Reply:
ok